src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -116,6 +117,26 @@ public void TestEmptyPush()
             RunReplication(config, 0, 0);
         }
 
+        [Fact]
+        public void TestPushDocWithFilter()
+        {
+            using (var doc1 = new MutableDocument("doc1"))
```

Comment:
A better test would be to set up a database with two documents, and set up a filter that rejects one of them.  Then you can check that only one document exists in the target database (`_otherDB`).

Reply:
you are right. updated.